data-access-layer/db/JobDetailSchema.js

```diff
@@ -45,5 +46,18 @@ const JobSchema = new Mongoose.Schema({
   }
 });
 
```

Comment:
Add index for https://github.com/cloudfoundry-incubator/service-fabrik-broker/blob/master/reports/BackupReportManager.js#L164-L167